Fix typo in compaq segment naming.

Apply with:
cd /usr/src/evms-2.5.3
patch -p1 < compaq_segments.patch
make install

--- evms-2.5.3a/plugins/dos/segs.c	10 Jun 2005 13:20:19 -0000
+++ evms-2.5.3b/plugins/dos/segs.c	12 Jun 2005 12:52:02 -0000
@@ -211,7 +211,7 @@
 				char junk1[64];
 				unsigned int junk2, junk3;
 
-				rc = sscanf(name, "%63[^/]c%ud%u",
+				rc = sscanf(name, "%63[^/]/c%ud%u",
 					    junk1, &junk2, &junk3);
 				if (rc == 3) {
 					strcat(name, "p");
